client/src/components/TrackItem.vue

```diff
@@ -42,6 +42,7 @@ export default {
       dense
       hide-details
       :input-value="inputValue"
+      color="white"
```

Comment:
This may be out of scope for this PR.  But should we support less locked down colors. including references to `this.$vuetify.theme.themes.dark.accent`  so there are less problems in the future if we decide to use a light theme?

Reply:
done